packages/runtime-core/src/helpers/renderSlot.ts

```diff
@@ -76,11 +76,11 @@ export function renderSlot(
     Fragment,
     {
       key:
-        (props.key ||
-          // slot content array of a dynamic conditional slot may have a branch
-          // key attached in the `createSlots` helper, respect that
-          (validSlotContent && (validSlotContent as any).key) ||
-          `_${name}`) +
+        (props.key == null || props.key === ''
+          ? // slot content array of a dynamic conditional slot may have a branch
+            // key attached in the `createSlots` helper, respect that
+            (validSlotContent && (validSlotContent as any).key) || `_${name}`
```

Comment:
`(validSlotContent && (validSlotContent as any).key)` 
This expression may also get a Symbol value

Reply:
```ts
const key = (props.key ||
          // slot content array of a dynamic conditional slot may have a branch
          // key attached in the `createSlots` helper, respect that
          (validSlotContent && (validSlotContent as any).key)

//...
key: (key && isSymbol(key)? String(key): `_${name}`) + //...
```